core/platform/image/gae_image_services.py

```diff
@@ -79,9 +80,13 @@ def compress_image(image_content, scaling_factor):
                     MAX_RESIZE_DIMENSION_PX, float(max(width, height))))
             new_width = int(width * new_scaling_factor)
             new_height = int(height * new_scaling_factor)
-        return images.resize(
-            image_data=image_content,
-            width=min(new_width, MAX_RESIZE_DIMENSION_PX),
-            height=min(new_height, MAX_RESIZE_DIMENSION_PX))
+        maxsize = (min(new_width, MAX_RESIZE_DIMENSION_PX),
+                   min(new_height, MAX_RESIZE_DIMENSION_PX))
+        img.thumbnail(maxsize, Image.ANTIALIAS)
+        with io.BytesIO() as output:
+            img.save(output, format=img_format)
+            contents = output.getvalue()
```

Comment:
new_image_content perhaps (both for parity with the existing name below (line 92) and to be clear about which image is being referred to here)

Reply:
Done